src/sql/workbench/contrib/objectExplorer/browser/serverTreeView.ts

```diff
@@ -196,6 +196,11 @@ export class ServerTreeView extends Disposable implements IServerTreeView {
 				this.deleteObjectExplorerNodeAndRefreshTree(connectionParams.connectionProfile).catch(errors.onUnexpectedError);
 			}
 		}));
+		this._register(this._configurationService.onDidChangeConfiguration(e => {
+			if (e.affectsConfiguration('datasource.connectionSort')) {
```

Comment:
Should use `CONNECTION_SORT_CONFIG_KEY` here

Reply:
updated